src/Spryker/Client/OauthPermission/OauthPermission/OauthPermissionReader.php

```diff
@@ -83,12 +83,22 @@ protected function extractPermissionsFromOauthToken(OauthAccessTokenDataTransfer
     }
 
     /**
-     * @param string $authorizationToken
+     * @param string|null $authorizationToken
      *
-     * @return array
+     * @return string|null
      */
-    protected function extractToken(string $authorizationToken): array
+    protected function extractToken(?string $authorizationToken): ?string
     {
-        return preg_split('/\s+/', $authorizationToken);
+        if (empty($authorizationToken)) {
```

Comment:
```suggestion
        if (!$authorizationToken) {
```
is per new coding standard

@SriRamyaVadlamudi Can we please schedule this as part of our bugfix process?